packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -195,16 +195,14 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
         fileMetadata={message['attachedFilesMetadata'] || []}
         downloadHandler={fileDownloadHandler}
         /* @conditional-compile-remove(file-sharing) */
-        strings={{ downloadFile: props.strings.downloadFile ?? locale.strings.messageThread.downloadFile }}
+        strings={{ downloadFile: props.strings.downloadFile, fileCardGroupMessage: props.strings.fileCardGroupMessage }}
```

Comment:
nit. Can we use a syntax similar to this const { strings } = props;
so then we can just say fileCardGroupmessage: strings.fileCardGroupMessage?

Reply:
Good catch!